metatensor-core/include/metatensor.h

```diff
@@ -1053,7 +1053,7 @@ mts_status_t mts_block_save_buffer(uint8_t **buffer,
  * The memory allocated by this function should be released using
  * `mts_tensormap_free`.
  *
- * `TensorMap` are serialized using numpy's `.npz` format, i.e. a ZIP file
+ * `TensorMap` are serialized using the `.mts` format, i.e. a ZIP file
```

Comment:
I think we can leave this mentioning NPZ, and saying we are encouraging the use of the mts extension to differentiate with arbitrary npz.